propose product ops role (#12327)
fleetdm/fleet: handbook/company/product-ops.md

> # 🎬 Product Ops

Author: ```suggestion
# ⚗️ Product Ops
```

## Let's start with why we exist. 📡

Ever wondered if your employer is monitoring your work computer?

Organizations make huge investments every year to keep their laptops and servers online, secure, compliant, and usable from anywhere. This is called "device management".

At Fleet, we think it's time device management became [transparent](https://fleetdm.com/transparency) and [open source](https://fleetdm.com/handbook/company#open-source).

## About the company 🌈

You can read more about the company in our [handbook](https://fleetdm.com/handbook/company), which is public and open to the world.

tldr; Fleet Device Management Inc. is a [recently-funded](https://techcrunch.com/2022/04/28/fleet-nabs-20m-to-enable-enterprises-to-manage-their-devices/) Series A startup founded and backed by the same people who created osquery, the leading open source security agent. Today, osquery is installed on millions of laptops and servers, and it is especially popular with [enterprise IT and security teams](https://www.linuxfoundation.org/press/press-release/the-linux-foundation-announces-intent-to-form-new-foundation-to-support-osquery-community).

## About this role 🔎

> Fleet is hiring our first Product Operations (Product Ops) team member to increase the team’s efficiency and coordination with other departments. A successful Product Ops will serve as an enabling force for the product team: removing time-consuming output-focused work from product managers, so they can focus on delivering value to customers and outcomes for the business.

Author: ```suggestion
Fleet is hiring our first Product Operations (Product Ops) team member to increase the team’s efficiency and coordination with other departments. A successful Product Ops contributor will serve as an enabling force for the product team: removing time-consuming, output-focused work from product managers so they can focus on delivering value to customers and outcomes for the business.
```

> Over time, this Product Ops will grow to:

Author: ```suggestion
Over time, this Product Ops role will expand to:
```

> - Recommend and implement new processes that improve the efficiency of the team, the quality of the team’s output, and the team communication with internal and external partners

Author: ```suggestion
- Recommend and implement new processes that improve the efficiency of the team, the quality of the team’s output, and the team's communication with internal and external partners
```

- Become an expert in the product processes of a high-performing team
- Ensure timely and high-quality outputs from the product team
- Understand the product and customers with sufficient depth to serve as secondary/backup point of contact
- Enable customer and user data collection and synthesis
- Contribute to improvements in the product

## Your primary responsibilities 🔭

> - Execute product processes as described in fleetdm.com/handbook/product 

Author: ```suggestion
- Execute product processes as described in https://fleetdm.com/handbook/product 
```

- Recommend and define improvements to product processes and tools
- Maintain and update the product section of the handbook
- Implement and drive change management for any new or modified product processes and tools across the team and/or the organization

> - Collect and report product team KPI metrics

Author: ```suggestion
- Collect and report product team KPI
```

- Ensure product team outputs are consistent and meets agreed-upon handoff requirements for other departments
- Design, implement, and execute systems for the product team to enable engineering, customer experience, marketing, sales and other teams
- Facilitate the collection and synthesis of qualitative and quantitative customer and user data
- Assist in organizing betas for larger product initiatives and feature releases
- Assist in improving the product where needed

## Are you our new team member? 🧑‍🚀
If most of these qualities sound like you, we would love to chat and see if we're a good fit.
- Experience with an engineering process such as Scrum/Agile
- Detail-oriented and highly organized
- Demonstrated ability to learn technical concepts
- Strong writing and oral communication for both general and technical topics
- Capable of translating technical concepts for a broader audience and vice versa
- Experience in communicating with external stakeholders 

## Why should you join us? 🛸

Learn more about the company and [why you should join us here](https://fleetdm.com/handbook/company#is-it-any-good).

## Want to join the team?

Want to join the team?

> Reach out to Mo Zhu on linkedin.

Reviewer: LinkedIn

Reviewer: `to [Mo Zhu on LinkedIn](https://www.linkedin.com/in/mo-zhu/)`
